website/content/posts/a-taste-of-policies.md

```diff
@@ -9,6 +9,8 @@ post](https://enterprisecontract.dev/posts/introducing-the-enterprise-contract/)
 basic concepts of the Enterprise Contract. This time, we explore it further to showcase the usage of
 policies.
 
+<!--more-->
+
```

Comment:
The PR is not in draft, is this intentional?

Reply:
Notice shorter excerpt at https://7a33b64c.enterprise-contract.pages.dev/posts/ vs https://enterprisecontract.dev/posts/ .